### DISCUSSION TITLE
Welcome to dmt-meshes Discussions! Please share your findings

### DISCUSSION THREAD

**Discussion by mosesalan** [Firework-Games-AI-Division/dmt-meshes#2] [Announcements]
Share your findings! so we can all discover!

## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AugmentedRealityCat** [comment]
First, thanks a lot for making this !
Here is a little video I made from my first "good enough" model after many experiments to better understand how this was all working.

https://user-images.githubusercontent.com/114955776/213616386-cdd1317d-8d7c-42be-808b-04d47c340a82.mp4

The model itself was synthesized using DMT-Meshes from the following parameters:
text prompt: `Tyrannosaurus Rex`
Image to PC model: `1B model`
PC to mesh method: `OpenAI`
Steps: `10 000`
Mesh rendering frequency: `500`
Step Preview: `None` 
Mesh learning rate: `0.0005`
Regulization weight:` 0.30`
Gridres: `128`
Multires: `4`
Run Mode: `Text to Mesh`

And here are pictures of the raw poly mesh, and of the textured version after using the wonderful [Dream Texture](https://github.com/carson-katri/dream-textures) add-on from the equally wonderful @carson-katri  


![tyragui](https://user-images.githubusercontent.com/114955776/213618391-2340c0b9-aa35-4f7a-b385-e0c34a78f171.png)
![tyragui2](https://user-images.githubusercontent.com/114955776/213618400-d9ee9357-54a3-4ce3-8b87-f66a5404bc18.png)

**Reply by AugmentedRealityCat** [reply]
Follow-up: I discovered I had been missing a key piece of information. Basically, I produced like a hundred models and ranked them and grouped them according to different parameters I was setting, because I wanted to learn what each parameter meant and how to use them. And my conclusion was that it did not make any sense. And in a sense I was right: all those parameters I was playing with, all but the resolution parameter in fact, are reserved exclusively for use with the DMTET method, and I never used that method in my tests, I only used OpenAI ! So all those parameters I was playing with methodically were doing nothing at all, and all I was observing was the natural variation from the text2pointcloud part, where I suppose there is a random seed or something similar varying the results even if you reuse the exact same text as a prompt.

Just in case you would miss that very important piece of information, and other important instructions, here is the link:
https://github.com/Firework-Games-AI-Division/dmt-meshes/blob/0.0.1/docs/MESH_GENERATION.md#advanced

And here is a copy of the info I had been missing (I thought the readme was all there was to read !)


> **Advanced**
> 
> Image to point cloud model - Pre-trained openAI model to generate mesh from an image.
> 
>  Point cloud to mesh model - You can use either point cloud to mesh implementation from openAI or [DMTet](https://github.com/NVIDIAGameWorks/kaolin/blob/master/examples/tutorial/dmtet_tutorial.ipynb).
> 
> Steps, mesh rendering, mesh learning rate, regularization, gridres, multires - parameters for point cloud to mesh. Except gridres, other parameters are only for DMTet. We will revamp the UI in the next release. For more information about the parameters, please visit [here](https://github.com/NVIDIAGameWorks/kaolin/blob/master/examples/tutorial/dmtet_tutorial.ipynb).
> 
> NOTE: DMTet mode only supports gridres=128

Hopefully this will be useful for someone !
